qiskit/visualization/counts_visualization.py

```diff
@@ -57,18 +56,6 @@ def _is_deprecated_data_format(data) -> bool:
     return False
 
 
-@deprecate_arg(
-    "data",
-    deprecation_description=(
-        "Using plot_histogram() ``data`` argument with QuasiDistribution, ProbDistribution, or a "
-        "distribution dictionary"
-    ),
-    since="0.22.0",
-    additional_msg="Instead, use ``plot_distribution()``.",
-    predicate=_is_deprecated_data_format,
-    pending=True,
```

Comment:
Is there a related PR for 0.46 that correctly deprecates this argument? So far it's only been pending deprecation, right? 🙂